BaseApp/PAG99999.txt

```diff
@@ -0,0 +1,108 @@
+OBJECT Page 99999 Camera
+{
+  OBJECT-PROPERTIES
+  {
+    Date=17-10-20;
+    Time=16:17:10;
+    Modified=Yes;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    Editable=No;
+    CaptionML=ENU=Camera;
+    SourceTable=Table99998;
+    PageType=StandardDialog;
+    SourceTableTemporary=Yes;
+    OnOpenPage=BEGIN
+                 CameraAvailable := CameraProvider.IsAvailable();
+
+                 IF CameraAvailable THEN BEGIN
```

Comment:
Better do 
IF NOT CameraAvailable THEN 
  EXIT;

Reply:
Done.